thirdparty/liblzma/liblzma.patch

```diff
@@ -0,0 +1,118 @@
+diff -rupN orig/CMakeLists.txt patched/CMakeLists.txt
+--- orig/CMakeLists.txt	1970-01-01 01:00:00.000000000 +0100
++++ patched/CMakeLists.txt	2020-03-31 21:52:39.000000000 +0200
+@@ -0,0 +1,114 @@
++cmake_minimum_required(VERSION 3.7)
++
++project(liblzma)
++
++set(SOURCES src/common/tuklib_cpucores.c
```

Comment:
Do we patch to only compile the subset of files/features we need?

Reply:
We can't run the configure script on Windows, and the only thing lzma provides for Windows are msbuild files which do not integrate well into our CMake build system, so I wrote a windows-specific CMakeLists.txt for the project.